### PR TITLE
[tw] Localize Virtualization for Traditional Chinese

### LINKED ISSUE #2310: [tw] Localize `Virtualization` for Traditional Chinese
1. Localize `Virtualization` for Traditional Chinese
2. https://glossary.cncf.io/virtualization/

### PR DESCRIPTION
**Pull request by johnlinp**
### Describe your changes

Origin content from https://glossary.cncf.io/virtualization/

### Related issue number or link (ex: `resolves #issue-number`)

resolves #2310

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by hwchiu — APPROVED** [review]
LGTM

File: content/zh-tw/virtualization.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+在雲端原生運算中，
+虛擬化指的是將一台實體電腦（有時或稱為伺服器），
+使其能夠運行多個獨立作業系統的方法。
+這些獨立的作業系統以及它們專屬的運算資源（CPU、記憶體和網路）被稱為虛擬機器。
+當我們提到[虛擬機器](/zh-tw/virtual-machine/)時，
+在講的其實是由軟體定義的電腦。
+這種電腦在使用上和與運作上都像是真實的電腦，
+但實際上它會與其他虛擬機器共用硬體資源。
+在[雲端運算](/zh-tw/cloud-computing/)裡，虛擬化技術會被大量地使用。
```
Comment (pichuang) [inline content/zh-tw/virtualization.md:18]:
```suggestion
[雲端運算](/zh-tw/cloud-computing/) 主要是由虛擬化技術所驅動。
```

**Review by ydFu — COMMENTED** [review]
Translate to '計算機' according [this discussion](https://github.com/cncf/glossary/pull/2020#discussion_r1191062566)  to our preference

**Comment by johnlinp** [comment]
@ydFu 我有接受你的修改建議了，再麻煩你看看，謝謝。

**Review by pichuang — APPROVED** [review]
lgmt

**Review by ydFu — APPROVED** [review]
LGTM,thanks @johnlinp

**Pull request merged**